### PR TITLE
chore(CHANGELOG): update to 5.1.0

### PR DESCRIPTION
**Pull request by Skaiir**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,83 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 5.1.0
+
+### Main changes
+
+* Multiselect context pad added to our tools for quick deletion, alignement and spacing
```
Comment (barmac) [inline CHANGELOG.md:13]:
```suggestion
* Multi-element context pad added to our tools for quick deletion, alignment, and spacing
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,83 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+* Telemetry migrated to Mixpanel
```
Comment (barmac) [inline CHANGELOG.md:14]: Not sure if this should be mentioned here. Our users don't benefit from the change.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,83 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+* `bpmn-js` and `dmn-js` have received UX improvements
```
Comment (barmac) [inline CHANGELOG.md:15]:
```suggestion
* BPMN and DMN editors have received UX improvements:
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,83 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+### General
+
+* `FEAT`: mask non-boolean flag values for ping event ([#2963](https://github.com/camunda/camunda-modeler/pull/2963))
+* `FEAT`: track set flags via ping event ([#2963](https://github.com/camunda/camunda-modeler/pull/2963))
+* `FEAT`: use @camunda/linting ([#2976](https://github.com/camunda/camunda-modeler/pull/2976))
+* `FEAT`: implement mixpanel telemetry ([#2934](https://github.com/camunda/camunda-modeler/issues/2934), [fec6ee4](https://github.com/camunda/camunda-modeler/commit/fec6ee4))
+* `FEAT`: track diagram deployments in mixpanel ([#2934](https://github.com/camunda/camunda-modeler/issues/2934), [a3e48b0](https://github.com/camunda/camunda-modeler/commit/a3e48b0))
+* `FEAT`: track link and button clicks in mixpanel ([#2934](https://github.com/camunda/camunda-modeler/issues/2934), [bbf0914](https://github.com/camunda/camunda-modeler/commit/bbf0914))
+* `FEAT`: track overlay usage in mixpanel ([#2934](https://github.com/camunda/camunda-modeler/issues/2934), [4e09d2e](https://github.com/camunda/camunda-modeler/commit/4e09d2e))
+* `FIX`: Debounced input fields no longer lose their values ([#2990](https://github.com/camunda/camunda-modeler/issues/2990))
```
Comment (barmac) [inline CHANGELOG.md:30]: Mixing lowercase and sentence-cased entries.

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,83 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+* `FIX`: cancel direct editing before shape deletion ([bpmn-js#1677](https://github.com/bpmn-io/bpmn-js/issues/1677))
+* `DEPS`: bump camunda-bpmn-js to 0.15.2 ([bc8b74870](https://github.com/camunda/camunda-modeler/commit/bc8b74870), [dbe78ef45](https://github.com/camunda/camunda-modeler/commit/dbe78ef45))
+* `DEPS`: bump bpmn-js-properties-panel to 1.2.0 ([bc8b74870](https://github.com/camunda/camunda-modeler/commit/bc8b74870))
+* `DEPS`: bump bpmn-js to 9.3.1 ([bc8b74870](https://github.com/camunda/camunda-modeler/commit/bc8b74870))
+* `DEPS`: bump bpmnlint-plugin-camunda-compat to 0.9.1 ([#2927](https://github.com/camunda/camunda-modeler/issues/2927), [d6cc308](https://github.com/camunda/camunda-modeler/commit/d6cc308))
+
+### DMN
+
+* `FEAT`: rework diagram interaction handles ([diagram-js#640](https://github.com/bpmn-io/diagram-js/pull/640))
+* `FEAT`: rework select and hover interaction on the diagram ([diagram-js#640](https://github.com/bpmn-io/diagram-js/pull/640), [diagram-js#643](https://github.com/bpmn-io/diagram-js/pull/643))
```
Comment (barmac) [inline CHANGELOG.md:75]:
```suggestion
* `FEAT`: rework select and hover interaction on the diagram ([diagram-js#640](https://github.com/bpmn-io/diagram-js/pull/640), [diagram-js#643](https://github.com/bpmn-io/diagram-js/pull/643))
* `FIX`: allow to scroll variable type in literal expression ([#2908](https://github.com/camunda/camunda-modeler/issues/2908))
```

**Review by barmac — CHANGES_REQUESTED** [review]
I left some suggestions and comments to address before we merge this.

**Comment by barmac** [comment]
This release is huuuuge 🤩

File: CHANGELOG.md [hunk 6]
```diff
@@ -6,6 +6,84 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+
+### Main changes
+
+* Multi-element context pad added to our tools for quick deletion, alignment, and spacing
+* Telemetry migrated to Mixpanel
+* BPMN and DMN editors have received UX improvements:
+	* clearer selection and multiselection
+	* input fields in the properties panel no longer lose changes
+* Conditional element template properties available in C8 
+* Stability and QOL improvements across the application
```
Comment (barmac) [inline CHANGELOG.md:19]: Not sure what QOL is. Also, is this about any specific changes? Let's make it explicit or remove it.

File: CHANGELOG.md (hunk 2 above)
Comment (nikku) [reply CHANGELOG.md:14]: What we sometimes do is to add a one line summary in the beginning of the release (cf. https://github.com/bpmn-io/diagram-js/blob/develop/CHANGELOG.md#850). I'd encourage you to drop the "Main changes" section in favor of the existing pattern.

**Review by barmac — APPROVED** [review]
✅

**Pull request merged**